### PR TITLE
Studio user movies

### PR DESCRIPTION
**Pull request by pranay2401**
# CHANGELOG
* Added users' movies support on studio
* Minor UI fixes on OTT

### CODE REVIEW

File: packages/im-ott/src/organisms/filmList/FilmList.js [hunk 1]
```diff
@@ -126,9 +126,7 @@ const FilmList = (props) => {
           {label} &nbsp; <RightOutlined />
         </div>
         <div className={styles.seeMore} onClick={handleSeeAll}>
-          <p>
-            See all <RightOutlined />
-          </p>
+          <p>See all</p>
```
Comment (ukirderohit) [inline packages/im-ott/src/organisms/filmList/FilmList.js:129]: why remove > ? 
it was looking good right

File: packages/im-ott/src/organisms/filmList/filmlist.module.scss [hunk 2]
```diff
@@ -2,6 +2,7 @@
   display: flex;
   flex-direction: row;
   max-height: 32rem;
+  // width: 100%;
```
Comment (ukirderohit) [inline packages/im-ott/src/organisms/filmList/filmlist.module.scss:5]: remove comment

File: packages/im-studio/src/graphql/queries.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { gql } from "@apollo/client";
 
 const QUERY_ALL_MOVIES = gql`
-  query {
-    movies {
+  query getAllMovies($userId: String) {
+    movies(userId: $userId) {
       id
```
Comment (ukirderohit) [inline packages/im-studio/src/graphql/queries.js]: dont use queries inside studio ,use inside imbase
check ott too if the change is not affecting other changes

File: packages/im-studio/src/organisms/appSkeleton/appskeleton.module.scss [hunk 4]
```diff
@@ -11,13 +14,14 @@
 
   .content {
     color: #000;
-    min-height: 40rem;
+    // padding-bottom: 4rem;
```
Comment (ukirderohit) [inline packages/im-studio/src/organisms/appSkeleton/appskeleton.module.scss:17]: comment

File: packages/im-studio/src/organisms/appSkeleton/appskeleton.module.scss [hunk 5]
```diff
@@ -11,13 +14,14 @@
...
+    overflow: auto;
   }
 
   .footer {
-    text-align: center;
-    background-color: black;
-    color: white;
-    box-shadow: 1rem 1rem 1rem #fff;
+    height: 4.8rem;
+    padding: 0rem;
```
Comment (ukirderohit) [inline packages/im-studio/src/organisms/appSkeleton/appskeleton.module.scss]: padding: 0
no rem needed

**Review by ukirderohit — CHANGES_REQUESTED** [review]

File: packages/im-ott/src/organisms/filmList/FilmList.js (hunk 1 above)
Comment (pranay2401) [reply packages/im-ott/src/organisms/filmList/FilmList.js:129]: Removed only the arrow key. Was not looking good with See all

File: packages/im-studio/src/graphql/queries.js (hunk 3 above)
Comment (pranay2401) [reply packages/im-studio/src/graphql/queries.js]: Updated

File: packages/im-studio/src/organisms/appSkeleton/appskeleton.module.scss (hunk 5 above)
Comment (pranay2401) [reply packages/im-studio/src/organisms/appSkeleton/appskeleton.module.scss]: Updated

**Review by ukirderohit — APPROVED** [review]

**Pull request merged**